### PR TITLE
itest: extend whitelist 2

### PR DESCRIPTION
**Pull request by joostjager**
Continuation of #4249

Reply in comments with everything that is encountered.

### CODE REVIEW

**Comment by joostjager** [comment]
Currently there is only a newly introduced error for the addition of `HtlcSubscribeEvents` to the itest in #4234

**Comment by joostjager** [comment]
We could consider lowering the catch-all rpc error logging to debug level warning? Returning errors over RPC is often not a system-level failure. Or only warning-log `context.Canceled` and a to be created generic shutting down error instance.

File: lntest/itest/log_error_whitelist.txt [hunk 1]
```diff
@@ -71,6 +71,7 @@
 <time> [ERR] HSWC: ChannelLink(<chan>): failing link: unable to synchronize channel states: unable to send chan sync message for ChannelPoint(<chan_point>): set tcp <ip>: use of closed network connection with error: unable to resume channel, recovery required
 <time> [ERR] HSWC: ChannelLink(<chan>): failing link: unable to synchronize channel states: Unable to send chan sync message for ChannelPoint(<chan_point>): write tcp <ip>-><ip>: use of closed network connection with error: unable to resume channel, recovery required
 <time> [ERR] HSWC: ChannelLink(<chan>): failing link: unable to synchronize channel states: Unable to send chan sync message for ChannelPoint(<chan_point>): write tcp <ip>-><ip>: write: broken pipe with error: unable to resume channel, recovery required
+<time> [ERR] HSWC: ChannelLink(<chan>): failing link: unable to synchronize channel states: unable to send chan sync message for ChannelPoint(<chan_point>): write tcp <ip>-><ip>: write: connection reset by peer with error: unable to resume channel, recovery required
```
Comment (cfromknecht) [inline lntest/itest/log_error_whitelist.txt:74]: these should be split out from the README changes since these are blocking a stable build.

**Comment by Roasbeef** [comment]
Build for this PR fails with: 
```
<time> [ERR] FNDG: Unable to advance state(<chan_point>): failed adding to router graph: funding manager shutting down
```

In the mean time will merge PRs that pass everything but this final check for the integration tests.

**Comment by joostjager** [comment]
Added

**Pull request merged**